### DISCUSSION TITLE
Welcome to inkline Discussions!

### DISCUSSION THREAD

**Discussion by alexgrozav** [inkline/inkline#220] [General]
## 👋 Welcome!

We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by kfern** [comment]
Hello! I'm new to vue. I'm learning it using this library. It's awesome. Thanks for sharing

**Reply by alexgrozav** [reply]
Welcome! Really happy to hear you enjoy using Inkline! You're going to love what's coming in Inkline 3. :) 

What are you currently working on?

**Reply by kfern** [reply]
I have been using react for 5 years but I don't like it very much so I am learning vue. At the moment I am testing concepts.

**Reply by alexgrozav** [reply]
I've learned both Vue and React at the beginning, but I stuck with Vue. Concepts are much easier to grasp and better suited for someone who loves being creative, in my opinion.

**Reply by milaan-muc** [reply]
Hi! Also new to Vue. Was using Knockoutjs for many years. Inkline is awesome so far. Will you further improve this UI Framework?
I relly love that you have validation rules for forms.
I miss toaster/snackbar and you guys could provide more modals/dialog components.

Trying to build a portfolio website to manage my own bitcoins, stocks, etc.

**Reply by alexgrozav** [reply]
Welcome @milaan-muc! I'm currently working very hard to get Inkline 3 done (for Vue 3). There will lots and lots of improvements. I'm still about one month away from releasing it, but I promise it's worth the wait. :)
